### PR TITLE
Fix issue 7911

### PR DESCRIPTION
**Pull request by vilktor370**
- Fixed the doc error for [Gaussian Blur](https://github.com/pytorch/vision/issues/7911)
- Run build commands to ensure the code and doc builds.
- Run style checks as specified in the [CONTRIBUTING.md](https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md)
- No unit tests are run because only modified a comment.

cc @vfdev-5

### CODE REVIEW

**Comment by vilktor370** [comment]
### Test results


image

File: torchvision/transforms/transforms.py [hunk 1]
```diff
@@ -1760,7 +1760,7 @@ def __repr__(self) -> str:
 class GaussianBlur(torch.nn.Module):
     """Blurs image with randomly chosen Gaussian blur.
     If the image is torch Tensor, it is expected
-    to have [..., C, H, W] shape, where ... means an arbitrary number of leading dimensions.
+    to have [N, C, H, W] shape, where ... means an arbitrary number of leading dimensions.
```
Comment (pmeier) [inline torchvision/transforms/transforms.py:1763]: As is the comment is not correct:

1. Shape `[C, H, W]` is also allowed. A batch dimension is not required.
2. The second part of the sentence references "...", which no longer exists.

You could align the sentence with the one from `F.pad`

https://github.com/pytorch/vision/blob/4af683107f967d4d435be4180020989dd8c3019c/torchvision/transforms/functional.py#L498-L499

e.g.

```suggestion
    to have [..., C, H, W] shape, where ... means at most one leading dimension.
```

**Review by pmeier — CHANGES_REQUESTED** [review]
Thanks for the PR @vilktor370.

File: torchvision/transforms/transforms.py (hunk 1 above)
Comment (vilktor370) [reply torchvision/transforms/transforms.py]: @pmeier 

Thank you for reviewing! I have changed the text as you suggested.

**Review by pmeier — DISMISSED** [review]
Thanks @vilktor370!

**Review by pmeier — COMMENTED** [review]
@vilktor370 Sorry, I forgot that we need to fix the functional API as well:

https://github.com/pytorch/vision/blob/d80d5bd21c6ee677af50d39689ba7005645b7ef4/torchvision/transforms/functional.py#L1327-L1329

Could you apply the same patch there.

**Comment by vilktor370** [comment]
@pmeier 

I just pushed the changes for `functional.py`. Could you review it again?

**Pull request merged**